modules/ROOT/pages/service-mesh-overview-and-landing-page.adoc

```diff
@@ -15,30 +15,33 @@ To alleviate issues arising from differences in these microservices, Anypoint Se
 
-With Anypoint Service Mesh from MuleSoft you can connect your applications, data, and devices through APIs in an applications network. You can manage, govern and secure your application network within Anypoint Platform.
+With Anypoint Service Mesh, you can connect your applications, data, and devices through APIs in an applications network. You can manage, govern, and secure your application network using Anypoint Platform.
 
-Microservices might exist outside of the MuleSoft ecosystem, all built in different languages and deployed to different environments. Anypoint Service Mesh enables you to extend your application network to any type of microservice. 
+Microservices can exist outside of the MuleSoft ecosystem, built using different languages and deployed to different environments. Anypoint Service Mesh enables you to extend your application network to any type of microservice. 
 
-Most importantly, the Anypoint Service Mesh can perform these tasks across all services, independent of the technology upon which it is built. Developers can focus on building out the business logic within. You can now discover, manage and secure any type of microservice from a central control plane in the Anypoint Platform.
+Most importantly, Anypoint Service Mesh can perform these tasks across all services, independent of the technology upon which it is built. You can now discover, manage and secure any type of microservice from a central control plane in the Anypoint Platform.
 
+== Anypoint Service Mesh Architecture
 
+A service mesh is an independent architecture layer, which is encapsulated in a Kubernetes cluster. In a service mesh architecture, instead of the services directly communicating with one another, a sidecar proxy is used to perform the same job. This allows for providing security to your services. The manner in which these sidecars interact is governed by an overall mixer, where a user can apply different policies and security.
 
-== Anypoint Service Mesh Architecture
-A service mesh is an independent architecture layer, which is encapsulated in a Kubernetes cluster. In a service mesh architecture, instead of the services directly  communicating with one another, a sidecar proxy is used to perform the same job. This allows for providing security to your services. The manner in which these sidecars interact is governed by an overall mixer, where a user can apply different policies and security.
+The mixer is a highly modular and extensible component, whose key function is to provide a layer of abstraction over the different policy and telemetry backend systems. This enables applications, such as Anypoint Platform to allow the rest of Istio to be agnostic of the backend processing.
```

Comment:
This enables applications, such as Anypoint Platform to allow the rest of Istio to be agnostic of the backend processing. >
Because of the mixer, applications like Anypoint Platform allow the rest of Istio to be agnostic of the backend processing

Reply:
Because of this abstraction, applications such as Anypoint Platform, enable the rest of Istio to be agnostic of the backend processing.